config.py

```diff
@@ -26,6 +28,7 @@ def __init__(self):
             self.USER_NAME = config['REDDIT']['USER_NAME']
             self.SUBREDDIT = config['REDDIT']['SUBREDDIT']
             self.REMOVE_SUBMISSIONS = config['REDDIT']['REMOVE_SUBMISSIONS']
+            self.SEND_ARTIST_REPLY = config['REDDIT']['SEND_ARTIST_REPLY']
```

Comment:
Can you add it to the `config.ini.dist`?

Reply:
Done!